Documentation/WSL.md

```diff
@@ -0,0 +1,8 @@
+### **Building on Windows, Targeting Linux via WSL**
+
+WSL is the Windows Subsystem for Linux, which allows you to run Linux applications inside Windows. It is available in recent builds of Windows 10 (>19041) and in Windows 11. Because Babylon Native apps are GUI applications, you will need WSL 2. Follow [these instructions](https://docs.microsoft.com/en-us/windows/wsl/tutorials/gui-apps) to prepare a WSL environment with GUI support. Babylon Native has been tested on Ubuntu; other distributions may or may not work.
```

Comment:
nit: Stylistically, I think most of our documentation adds newlines semi-regularly throughout paragraphs to make paragraphs more readable in text form as well as formatted.